src/components/layout/CommentInput.jsx

```diff
@@ -18,11 +18,18 @@ const CommentInput = () => {
       setInput('');
       setMessage('');
       setFocus(false);
+      refreshPage();
     } else {
       setMessage('Please subscribe to comment');
     }
   };
 
+  const refreshPage = () => {
+    setTimeout(() => {
+      window.location.reload();
```

Comment:
No. Update  state and have React do what it does bed. window.location.reload is a hack.

Reply:
Fixed